modules/etc/commands/query.go

```diff
@@ -0,0 +1,44 @@
+package commands
+
+import (
+	"encoding/hex"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	cmn "github.com/tendermint/tmlibs/common"
+
+	"github.com/tendermint/basecoin/client/commands"
+	"github.com/tendermint/basecoin/client/commands/query"
+	"github.com/tendermint/basecoin/modules/etc"
+	"github.com/tendermint/basecoin/stack"
+)
+
+// EtcQueryCmd - command to query raw data
+var EtcQueryCmd = &cobra.Command{
+	Use:   "etc [key]",
+	Short: "Get data stored under key in etc",
+	RunE:  commands.RequireInit(etcQueryCmd),
+}
```

Comment:
I need naming help. 

It is like etcd that bucky refers to a lot. 
It is also like the merkleeyes app, thus eyes. 
Maybe a different name? 

I need help naming for sure